### PR TITLE
chore/update data dictionary for the VacTracker dataset

### PR DESCRIPTION
**Pull request by gkuffel**
Jira Ticket: [PXP-xxxx](https://ctds-planx.atlassian.net/browse/PXP-xxxx)
- [ ] Remove this line if you've changed the title to (PXP-xxxx): <title>

### Improvements
-Updated FHIR definitions
-Changed properties to align with FHIR, ICMJE, and clinicaltrials.gov props to enable interop

### CODE REVIEW

File: gdcdictionary/schemas/clinical_trials.yaml [hunk 1]
```diff
@@ -124,44 +132,49 @@ properties:
       - "Phase II/III"
       - "Phase N/A"
 
-  clinical_trials:
```
Comment (paulineribeyre) [inline gdcdictionary/schemas/clinical_trials.yaml:127]: breaking change - do we have no existing data with the `clinical_trials` prop?

File: gdcdictionary/schemas/clinical_trials.yaml [hunk 2]
```diff
@@ -124,44 +132,49 @@ properties:
...
+# ICMJE property http://www.icmje.org/recommendations/browse/publishing-and-editorial-issues/clinical-trial-registration.html
+  nct_number:
     description: >
-      Indication of clinical trial numbers associated with a study.
+      The organization's unique protocol identification number assigned to the clinical study.
     type: array
     items:
       type: string
 
-  status:
```
Comment (paulineribeyre) [inline gdcdictionary/schemas/clinical_trials.yaml:134]: same for `status`

File: gdcdictionary/schemas/clinical_trials.yaml [hunk 3]
```diff
@@ -124,44 +132,49 @@ properties:
...
 
+# VacTracker dataset specific property
   inprogress_clinical_trials:
     description: >
       In progress Clinical Trials
     type: array
     items:
       type: string
 
-  countries:
```
Comment (paulineribeyre) [inline gdcdictionary/schemas/clinical_trials.yaml:157]: same for `countries`

File: gdcdictionary/schemas/clinical_trials.yaml (hunk 1 above)
Comment (gkuffel) [reply gdcdictionary/schemas/clinical_trials.yaml:127]: We don't have any existing data with this prop. No data has been submitted to the clinical trials node yet.

**Review by paulineribeyre — APPROVED** [review]

**Review by svburke — APPROVED** [review]

**Review by hugheslaron — APPROVED** [review]

**Pull request merged**